dashboard/dashboard.xml

```diff
@@ -0,0 +1,452 @@
+<form version="1.1" theme="dark">
+  <label>SC4S Metrics and Events Dashboard</label>
+  <description>Monitor SC4S instances connected to this Splunk instance.</description>
+  <search id="baseMetricsSearch">
+    <query>
+      | mstats 
+      max("spl.sc4syslog.center.received.processed")
+      max("spl.sc4syslog.source.processed")
+      max("spl.sc4syslog.dst.dropped")
+      max("spl.sc4syslog.dst.queued")
+      prestats=true
+      WHERE "index"="_metrics"
+      BY sc4s_container
+      span=$span$
+    </query>
+    <earliest>$time_range.earliest$</earliest>
+    <latest>$time_range.latest$</latest>
+    <sampleRatio>1</sampleRatio>
+  </search>
+  <search id="baseEventsSearch">
+    <query>
+    index=* sc4s_container=$sc4s_instance$ 
+    | stats count by _time index sc4s_tags
+    </query>
+    <earliest>$time_range.earliest$</earliest>
+    <latest>$time_range.latest$</latest>
+    <sampleRatio>1</sampleRatio>
+  </search>
+  <fieldset submitButton="false"></fieldset>
+  <row>
+    <panel>
+      <input type="dropdown" token="span" searchWhenChanged="true">
+        <label>Span</label>
+        <choice value="30s">30s</choice>
+        <choice value="1m">1m</choice>
+        <choice value="3m">3m</choice>
+        <choice value="30m">30m</choice>
+        <choice value="1h">1h</choice>
+        <choice value="6h">6h</choice>
+        <choice value="12h">12h</choice>
+        <choice value="1d">1d</choice>
+        <default>30s</default>
+        <initialValue>30s</initialValue>
+      </input>
+      <input type="time" token="time_range">
+        <label>Timer</label>
+        <default>
+          <earliest>rt-15m</earliest>
+          <latest>rt</latest>
+        </default>
+      </input>
+      <html>
+        <p>Choose time interval and window.</p>
+        <p>By default, the dashboard loads metrics from every 30 seconds for the 15-minutes window.</p>
+        <p>By default, the dashboard loads events for the 15-minutes window if provided the SC4S instance name.</p>
+      </html>
+    </panel>
+  </row>
+  <row>
+    <panel>
+      <html>
+      <h1>
+        Metrics
+      </h1>
+      </html>
+    </panel>
+  </row>
+  <row>
+    <panel>
+      <title>Received Messages</title>
+      <html>
+        <p>Healthy SC4S instance logs the number of received messages each 30 seconds.</p>
+        <p>The number should grow by at least 1 each 30 seconds, because metrics message counts in as a received message.</p>
```

Comment:
<p>The number should grow by at least 1 every 30 seconds. This is because the metrics message counts as a received message.</p>